rl4co/utils/meta_trainer.py

```diff
@@ -0,0 +1,100 @@
+import lightning.pytorch as pl
+import torch
+import math
+import copy
+from torch.optim import Adam
+
+from lightning import Callback
+from rl4co import utils
+import random
+log = utils.get_pylogger(__name__)
+
+
+class ReptileCallback(Callback):
+
+    # Meta training framework for addressing the generalization issue
+    # Based on Zhou et al. (2023): https://arxiv.org/abs/2305.19587
+    def __init__(self, meta_params, print_log=True):
```

Comment:
[Minor] I think it would be more clear to list the Hyperparameters.

For  example

```python
def __init__(self, 
alpha=...,
alpha_decay=...
... 
print_log=True):
```

which is generally easier to maintain and to document.